specification/carbon/resource-manager/readme.md

```diff
@@ -0,0 +1,86 @@
+# Carbon
+
+> see https://aka.ms/autorest
+
+This is the AutoRest configuration file for Carbon.
+
+
+
+---
+## Getting Started
+To build the SDK for Carbon, simply [Install AutoRest](https://aka.ms/autorest/install) and in this folder, run:
+
+> `autorest readme.md`
+
+To see additional help and options, run:
+
+> `autorest --help`
+---
+
+## Configuration
+
+
+### Basic Information
+These are the global settings for the Carbon Optimization API.
+
+``` yaml
+title: CarbonOptimizationClient
+openapi-type: arm
+openapi-subtype: rpaas
+tag: package-2024-02-01-preview
+```
+
+### Tag: package-2023-04-01-preview
+
+These settings apply only when `--tag=package-2023-04-01-preview` is specified on the command line.
+
+``` yaml $(tag) == 'package-2023-04-01-preview'
+input-file:
+- Microsoft.Carbon/preview/2023-04-01-preview/main.json
```

Comment:
seems this folder does not exist in target branch?
https://github.com/Azure/azure-rest-api-specs/pull/27978/checks?check_run_id=22135682377